index.html

```diff
@@ -37,6 +37,148 @@ <h1 class="hero">I’m Hamdullah
+                <h2 class="card-title">Multi-Post
+                    Stories</h2>
+                <ul class="work">
+                    <li class="canopy">CANOPY</li>
+                    <img class="counter1" src="images/Counter.svg" alt="">
+                    <li class="back-end-dev">Back End Dev</li>
+                    <img class="counter2" src="images/Counter.svg" alt="">
+                    <li class="year">2015</li>
+                </ul>
+                <p class="card-details">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li class="html languages-text">html</li>
+                    <li class="css languages-text">css</li>
+                    <li class="javascript languages-text">javaScript</li>
+                </ul>
+                <button class="project-btn project-btn-text">See Project</button>
+            </section>
+
+            <section class="about">
+
+                <div class="about-part">    
+                    <h3 class="about-title">About <br>
+                        Myself
+                    </h3>
+                    <p class="about-details">
+                        Hello I’m a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.
+                    </p>
+                    <p class="about-connect">LET’S CONNECT</p>
+                    <ul class="about-link">
+                        <li class="social-links"><a href="https://twitter.com/HamdullahAndar"><img src="images/twitter.svg" alt="twitter"></a></li>
+                        <li class="social-links"><a href="https://www.linkedin.com/in/hamdullah-andar-98b35894/"><img src="images/Linkedin.svg" alt="linkedin"></a></li>
+                        <li class="social-links"><a href="https://mail.google.com"><img src="images/gmail.svg" alt="gmail"></a></li>
+                        <li class="social-links"><a href="https://github.com/Hamdullah-Andar"><img src="images/github.svg" alt="github"></a></li>
+                        <li class="social-links"><a href="https://vectorstock.com"><img src="images/vector.svg" alt="vector"></a></li>
+                    </ul>
+                    <button class="get-resume">Get my resume</button>    
+                </div>
+
+                <div class="languages-part">
+                    <ul class="all-skills">
+                        <li class="skills">
+                            <h3 class="about-languages">Languages</h3>
+                            <img src="images/down-arrow.svg" alt="">
+                        </li>
+                    </ul>
+
+                    <section class="all-languages">
```

Comment:
- [ ] Kindly align the languages with the title as aligned in the Figma design.

-   Your design
  ![Hami1](https://user-images.githubusercontent.com/87394385/213784990-22871f54-894b-404d-8101-919e94244d0c.JPG)

-   Figma design
  ![Hami2](https://user-images.githubusercontent.com/87394385/213785162-710d2205-15a7-4e00-95f2-a3e46b1bc0e0.JPG)

-  Please note that the languages have a slightly bigger margin than the title. Check the specified margins and padding from the design and apply them accordingly.